docs/misc/projects.rst

```diff
@@ -250,3 +250,17 @@ It enables solving environments involving partial observability or locomotion (e
 | Authors: Corentin Léger, Gautier Hamon, Eleni Nisioti, Xavier Hinaut, Clément Moulin-Frier
 | Github: https://github.com/corentinlger/ER-MRL
 | Paper: https://arxiv.org/abs/2312.06695
+
+FootstepNet Envs
+----------------
+
+Footsteps Planning RL Environments for Fast On-line Bipedal Footstep Planning and Forecasting.
```

Comment:
could you shorten the description and make it less "abstract-like"?
you can take inspiration from https://github.com/DLR-RM/stable-baselines3/pull/2059 ;)

Reply:
I made a shorter new version, tell me if it's okay for you ;)